RELEASING.md

```diff
@@ -8,10 +8,12 @@ At a high level, the process is as follows:
 
 1. The developer prepares the release blog post.
 1. The developer adds their private key/passphrase as repository secrets
+1. The developer generates a new sonatype User Token, installs it credentials into GitHub, installs it into their local maven settings for testing
```

Comment:
see my comment below, I'd set things up to the Kroxylicious Robot's Sona type was long-lived and stable.

Reply:
sounds good, I had a poke around the UI and could see the users associated with the namespace, how do I get added? edit.. and now it's returning 403, I think I've managed to blacklist my ip by exploring their UI :(